{{cookiecutter.project_slug}}/clients/vue3/src/views/Dashboard.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <div class="dashboard">
-    <h2>This is your dashboard</h2>
+  <div>
+    <h1 class="heading">This is your dashboard</h1>
```

Comment:
**Info:** Heading tags should start in hierarchy order, starting with `<h1>`. This helps users with screen readers and other assistive technology to appropriately find information on the page.

If specific is styling is needed for a heading tag, the CSS should be updated (vs using a different heading # tag).